tensorboard/plugins/graph/tf_graph_dashboard/tf-graph-dashboard.html

```diff
@@ -83,7 +84,8 @@
         out-graph="{{_graph}}"
         out-stats="{{_stats}}"
         progress="{{_progress}}"
-out-hierarchy-params="{{_hierarchyParams}}"
+        out-hierarchy-params="{{_hierarchyParams}}"
+        op-compat-feature-enabled="[[debuggerDataEnabled]]"
```

Comment:
I like this feature. What if we just enabled it by default? Just like how coloring by XLA clustering is enabled by default.

`debuggerDataEnabled` is only true if the debugger plugin backend detects events issued by the debugger. That boolean is no longer guarded by a URL GET parameter, so it is a bit irrelevant to this feature.